api/src/main/resources/workbench-api.yaml

```diff
@@ -4841,6 +4845,11 @@ definitions:
     - NONE
     - PERSON
     - SURVEY
+    - FITBIT
+    - FITBIT_HEART_RATE_SUMMARY
+    - FITBIT_HEART_RATE_LEVEL
+    - FITBIT_ACTIVITY
+    - FITBIT_INTRADAY_STEPS
     - BOTH
```

Comment:
So if i select multiple of these prepackaged concept sets, how does the request/API handle this? The request only carries an enum value currently.

Reply:
I will be handling multiple PRE-PACKAGED concept set in the ticket RW-5687 along with UI changes

Reply:
Gotcha